lib/milc_interface.cpp

```diff
@@ -878,14 +878,25 @@ void qudaInvert(int external_precision,
 
 #ifdef GPU_CLOVER_DIRAC
 
-void* qudaCreateExtendedGaugeField(void* gauge, int geometry, int precision)
+static inline void* createExtendedGaugeField(void* gauge, int geometry, int precision, int resident)
 {
   qudamilc_called<true>(__func__);
```

Comment:
Just wondering whether we should us `qudamilc_called<>` only in exposed functions. 
However, we can keep it this way for now and I will create a separate cleanup issue and we can decide whether we also want to have the `qudamilc_called<false>` call before exiting for all exposed functions.

But as this is only verbose output it does not prevent the merge. It just made me aware of another inconsistency in the milc interface.